SQL/0000-00-00-schema.sql

```diff
@@ -2044,3 +2044,13 @@ CREATE TABLE `candidate_consent_history` (
   `EntryStaff` varchar(255) DEFAULT NULL,
   CONSTRAINT `PK_candidate_consent_history` PRIMARY KEY (`CandidateConsentHistoryID`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+
+-- A table to store the hashed value of passwords that were mistakenly recored
+-- in the `history` table. These passwords will not be allowed to be used as
```

Comment:
I'm not going to block over it, but I don't think this comment is accurate. Passwords can be blacklisted for any reason you want.